assets/js/modules/adsense/datastore/sites.test.js

```diff
@@ -0,0 +1,218 @@
+	} );
+
+	afterEach( () => {
+		unsubscribeFromAll( registry );
+	} );
+
+	describe( 'actions', () => {} );
+
+	describe( 'selectors', () => {
+		describe( 'getSites', () => {
+			it( 'uses a resolver to make a network request', async () => {
+				fetchMock.getOnce(
+					/^\/google-site-kit\/v1\/modules\/adsense\/data\/sites/,
+					{ body: fixtures.sites, status: 200 }
+				);
+
+				const accountID = fixtures.clients[ 0 ]._accountID;
+
+				const initialSites = registry
+					.select( MODULES_ADSENSE )
+					.getSites( accountID );
+
+				expect( initialSites ).toEqual( undefined );
+				await subscribeUntil(
+					registry,
+					() =>
+						registry
+							.select( MODULES_ADSENSE )
+							.getSites( accountID ) !== undefined
+				);
+
+				const sites = registry
+					.select( MODULES_ADSENSE )
+					.getSites( accountID );
+
+				expect( fetchMock ).toHaveFetchedTimes( 1 );
+				expect( sites ).toEqual( fixtures.sites );
+			} );
+
+			it( 'does not make a network request if sites for this account are already present', async () => {
+				const accountID = fixtures.clients[ 0 ]._accountID;
+
+				// Load data into this store so there are matches for the data we're about to select,
+				// even though the selector hasn't fulfilled yet.
+				registry
+					.dispatch( MODULES_ADSENSE )
+					.receiveGetSites( fixtures.sites, { accountID } );
+
+				const sites = registry
+					.select( MODULES_ADSENSE )
+					.getSites( accountID );
+
+				await subscribeUntil( registry, () =>
+					registry
+						.select( MODULES_ADSENSE )
+						.hasFinishedResolution( 'getSites', [ accountID ] )
+				);
+
+				expect( fetchMock ).not.toHaveFetched();
+				expect( sites ).toEqual( fixtures.sites );
+			} );
+
+			it( 'dispatches an error if the request fails', async () => {
+				const response = {
+					code: 'internal_server_error',
+					message: 'Internal server error',
+					data: { status: 500 },
+				};
+				fetchMock.getOnce(
+					/^\/google-site-kit\/v1\/modules\/adsense\/data\/sites/,
+					{ body: response, status: 500 }
+				);
+
+				const fakeAccountID = 'pub-777888999';
+				registry.select( MODULES_ADSENSE ).getSites( fakeAccountID );
+				await subscribeUntil(
+					registry,
+					() =>
+						registry
+							.select( MODULES_ADSENSE )
+							.isFetchingGetSites( fakeAccountID ) === false
+				);
+
+				expect( fetchMock ).toHaveFetchedTimes( 1 );
+
+				const sites = registry
+					.select( MODULES_ADSENSE )
+					.getSites( fakeAccountID );
+				expect( sites ).toEqual( undefined );
+				expect( console ).toHaveErrored();
+			} );
+		} );
+		describe( 'getSite', () => {
+			beforeEach( () => {
+				const accountID = fixtures.clients[ 0 ]._accountID;
+
+				// Load data into this store so there are matches for the data we're about to select,
+				// even though the selector hasn't fulfilled yet.
+				registry
+					.dispatch( MODULES_ADSENSE )
+					.receiveGetSites( fixtures.sites, { accountID } );
+			} );
+			it.each( [
```

Comment:
Can you add a line break here?
```suggestion

			it.each( [
```